.dockerignore

```diff
@@ -0,0 +1,16 @@
+.vscode/
+_build/
```

Comment:
Maybe ignore all dirs which starts with ? (_*)?

Reply:
Added ignoring of all dirs starting with _